.github/workflows/infrautils-edx-deploy.yaml

```diff
@@ -0,0 +1,66 @@
+# This workflow installs infrautils (DevOps maintained infrastructure as service python package) from JFrog
+# and uses infrautils (boto3) to deploy a EDX via cloudformation stack update
+# https://github.com/energyhub/infra-utils
+name: Infrautils CFN Deploy
+
+on:
+  workflow_call:
+    secrets:
+      JFROG_CI_PASSWORD:
+        required: true
+    inputs:
+      service:
```

Comment:
this is unused

Reply:
> Not sure what the grand plan is here so forgive me if this is obvious, but why would you deploy edx from another repository besides edx?

The use case right now is some services need EDX to be deployed in parallel - so MEC is actually calling an EDX deploy in qa-6 as part of their CI/CD process now as well. This simplifies that so that instead of calling a Codebuild job and paying for both a gha runner and a codebuild container, we're not duplicating any effort and calling it directly from the gha runner.
Clearly microservices shouldn't deploy each other, so not a pattern we're continuing in the future, this is just for EDX specifically.

Reply:
got it ty!